apis/services/wishService.js

```diff
@@ -0,0 +1,43 @@
+const { wishDao } = require('../models');
+
+const createWish = async (productId, userId) => {
+  const [checkWishlist] = await wishDao.checkWishlist(productId, userId)
+  if (checkWishlist) {
+    const err = new Error(`ALREADY_EXIST`);
+    err.statusCode = 400;
+    throw err;
+  }
+  await wishDao.createWish(productId, userId)
+  const showWish = await wishDao.showWish(userId)
+  return showWish
+}
+
+const removeWish = async (productId, userId) => {
+  const [checkWishlist] = await wishDao.checkWishlist(productId, userId)
+  if (!checkWishlist) {
+    const err = new Error(`NOT_EXIST_IN_WISHLIST`);
+    err.statusCode = 400;
+    throw err;
+  }
+  await wishDao.removeWish(productId)
+  const showWish = await wishDao.showWish(userId)
+  if (!showWish[0]) {
+    return (`WISHLIST_IS_EMPTY`)
+  }
+  return showWish
+}
+
+const postWish = async (userId) => {
```

Comment:
좋아요 목록 가져오는 service인가요??
그렇다면 아래와 같이 수정하면 좋을 것 같네요.
```suggestion
const getWishList = async (userId) => {
```